pytorch_lightning/callbacks/gradient_accumulation_scheduler.py

```diff
@@ -3,6 +3,7 @@
 ====================
 
 Change gradient accumulation factor according to scheduling.
+This callback also calls ``optimizer.step()`` for the last indivisible batch.
```

Comment:
I think it should say the step number indivisible

Reply:
or rather be 'the last indivisible step number'?